frontend/public/components/conditions.tsx

```diff
@@ -1,8 +1,10 @@
 import * as React from 'react';
 import * as _ from 'lodash-es';
 
-export const Conditions: React.SFC<ConditionsProps> = ({obj}) => {
-  const conditions = _.get(obj, 'status.conditions');
+export const Conditions: React.SFC<ConditionsProps> = ({obj: conditions}) => {
+  if (!conditions.length) {
```

Comment:
Missing conditions should be handled below since we display "No conditions." If we want an empty array check, we should use `_.isEmpty()` below.